rmgpy/pdep/cse.pyx

```diff
@@ -343,7 +343,7 @@ def get_rate_coefficients_CSE_Advanced(network,T,P,neglect_high_energy_collision
 
 
     pvlmbd = np.linalg.solve(eigen_vectors,Pv)
-    Gvlmbd = eigen_vectors.T @ k2v
```

Comment:
Could you help me understand why is this necessary?

Reply:
I believe this is identical. So strictly speaking unnecessary XD.

Reply:
I think I got rid of this.

Reply:
Oh, could you squash that one commit that fixes this with this one?

Reply:
fixed